src/dune_engine/package.ml

```diff
@@ -756,21 +761,21 @@ let load_opam_file file name =
   let get_one name =
     let* opam = opam in
     let* value = Opam_file.get_field opam name in
-    match value with
-    | String (_, s) -> Some s
+    match value.pelem with
+    | String s -> Some s
     | _ -> None
   in
   let get_many name =
     let* opam = opam in
     let* value = Opam_file.get_field opam name in
-    match value with
-    | String (_, s) -> Some [ s ]
-    | List (_, l) ->
+    match value.pelem with
+    | String s -> Some [ s ]
+    | List l ->
       let+ l =
-        List.fold_left l ~init:(Some []) ~f:(fun acc v ->
+        List.fold_left l.pelem ~init:(Some []) ~f:(fun acc v ->
             let* acc = acc in
-            match v with
-            | OpamParserTypes.String (_, s) -> Some (s :: acc)
+            match v.OpamParserTypes.FullPos.pelem with
```

Comment:
We don't qualify module names in fields. So this should be `v.pelem`. If necessary, you can add a type annotation to `v`.